src/domain/users/index.types.d.ts

```diff
@@ -81,3 +80,12 @@ interface IUsersRepository {
   persistNew({ phone, phoneMetadata }: NewUserInfo): Promise<User | RepositoryError>
   update(user: User): Promise<User | RepositoryError>
 }
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+interface ReadonlyArray<T> {
```

Comment:
if this is a generic implementation probably should be in primitives. this is not a blocker to merge this PR